src/rnnodeapp/index.js

```diff
@@ -62,7 +62,8 @@ function start() {
   const route = Router(db, media, {
     host: HOST,
     media: { mode: 'push' },
-    staticRoot: STATIC_PATH
+    staticRoot: STATIC_PATH,
+    writeFormat: 'osm-p2p-syncfile'
```

Comment:
could we do `sync: { writeFormat: 'osm-p2p-syncfile' }` instead? same on your mapeo-desktop PR -- same way we're namespacing the `media` opts

Reply:
sure sounds good!